pawan/html/practice_assignmenr/Programming_Exercises4.html

```diff
@@ -0,0 +1,61 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <title>Resume</title>
+        <style> 
+        table , th , td {
+        border: 2px solid black;}
+        </style>
+    </head>
+    <body>
+        <h1>Pawan Sharad Jagtap</h1>
+            <p><strong>pavanj10may@gmail.com</strong></p>
+            <hr>
+        <h3>*About Me*</h3>
+            <p>My name is <i>Pawan Sharad jagtap</i>.I'm 23 years old. I'm from Pune.<br> currently pursuing my degree in <em>BCS</em> from swaraj college of science and computer studies. </p>
+        <h3>Work</h3>
+            <p>
+            <b><em>I'm Fresher</em></b> 
+            </p>
+        <h3>To Do List </h3>
+            <a href="todolist.html" target="_blank">Click here </a>
+        <h3>Stuff I would like to do someday</h3>
+            <a href="StuffIwouldliketodosomeday.html" target="_blank">CLick here </a>
+        <h3>Interest</h3>
+            <ul  style="list-style-type:square";>
```

Comment:
remove one space after "ul" tag, also correct the inline style like this,
style="list-style-type: square;"
you wrote the wrong syntax